styles.css

```diff
@@ -387,3 +387,106 @@ hr {
   line-height: 43px;
   color: #fff;
 }
+
+.contact-section {
+  width: 87%;
+  padding: 120px 0;
+  margin: auto;
+  display: flex;
+  flex-direction: column;
+}
+
+.contact-section .contact-head {
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 32px;
+  line-height: 44px;
+  text-align: center;
+  color: #172b4d;
+  align-self: center;
+}
+
+.contact-section .form {
+  display: grid;
+  grid-template-columns: 1fr;
+  grid-template-rows: auto auto auto auto;
+  gap: 20px;
+  margin-top: 40px;
+  align-items: center;
+}
+
+.contact-section .form .form-input {
+  border-bottom: 3px solid #dbd8d7;
+  height: 40px;
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 15px;
+  line-height: 24px;
+  display: flex;
+  align-items: center;
+  border: none;
+  padding: 12px;
+}
+
+.contact-section .form textarea {
+  background-color: #fbf8f7;
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 15px;
+  line-height: 24px;
+  color: #3c3a39;
+  padding: 12px;
+}
+
+.contact-section .form .send {
+  padding: 12px;
+  gap: 10px;
+  width: 131px;
+  height: 48px;
+  background-color: #ff6b00;
+  margin: auto;
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 17px;
+  line-height: 24px;
+  letter-spacing: 0.03em;
+  color: #fff;
+}
+
+.contact-section .form .send:hover {
+  background: rgba(0, 0, 0, 0.3);
+  background-color: #fff5e1;
+  border: 1px solid #f55800;
+  color: #f55800;
+}
+
+.contact-section hr {
+  margin: 100px auto 0 auto;
+  width: 100%;
+  background-color: #dfe1e6;
+}
```

Comment:
- [ ] As you can see 😎 in the Figma template design, the `hr` covers 100% width of the page. In addition, the social icons are close to the `hr`. Therefore as a way of sticking to the Figma design including the font, colors, images, and spacing (margin/padding), I kindly request make sure that you change the requested changes to match the Figma design 🥰 .

Your design 😎 
![image](https://user-images.githubusercontent.com/99394374/220354528-f9095ab1-656a-42b2-b60c-e6245fc7f66f.png)


Figma design 😎 
![image](https://user-images.githubusercontent.com/99394374/220354425-b289b0b3-7667-4b15-8a5f-99a6ba6c600e.png)